Scripts/Engine/LuaUtil.lua

```diff
@@ -0,0 +1,888 @@
+-----<style>table.function_list td.name {min-width: 370px;}</style>
+--- Lua support functions to simplify operations in scripts. To use, include the module with:
+---	local LuaUtil = require("Engine.LuaUtil")
+--- @luautil LuaUtil
+
+local Type= require("Engine.Type")
+local LuaUtil = {}
+local operators = {
+    function(a, b) return a == b end,
+    function(a, b) return a ~= b end,
+    function(a, b) return a < b end,
+    function(a, b) return a <= b end,
+    function(a, b) return a > b end,
+    function(a, b) return a >= b end,
+}
+
+-- Helper function for ping-pong calculation on a single component
+local function pingPongComponent(t, length)
```

Comment:
Please use CamelCase everywhere, according to coding conventions. Also, what does "ping-pong" specifically mean here?